src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,59 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String CSV_SEPARATOR = ",";
+    private static final String CSV_OPERATION_TYPE = "buy";
+    private static final int CSV_ROW_ELEMENT = 0;
+    private static final int CSV_ROW_DATA_ELEMENT = 1;
```

Comment:
I'd go for something like AMOUNT_INDEX, OPERATION_INDEX to make it more informative